benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -764,8 +904,8 @@ void GraphicsBenchmarkApp::RecordCommandBufferSkybox(PerFrame& frame)
     frame.cmd->BindIndexBuffer(mSkyBox.mesh);
     frame.cmd->BindVertexBuffers(mSkyBox.mesh);
     {
-        SkyboxData data = {};
-        data.MVP        = mCamera.GetViewProjectionMatrix() * glm::scale(float3(500.0f, 500.0f, 500.0f));
+        SkyboxData data{};
```

Comment:
Nit: Is there a reason this was changed from `SkyboxData data = {}` ? I thought that one was recommended syntax for initializing structures. go/totw/88#best-practices-for-initialization

Reply:
oops, changed by mistake, reverted